mantidproject/mantid
# LoadLamp made parameter loading optional

This makes the loading of sample logs optional in LoadLamp.

**To test:**
Use LoadLamp to load the attached file (remove .zip extension first).
There should be no error in doing this.

[15Jan_up_LAMP.hdf.zip](https://github.com/mantidproject/mantid/files/1631255/15Jan_up_LAMP.hdf.zip)



No issue.

**Release Notes** 

No need for release notes, since this is a bug fix in a recently introduced algorithm.


#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
